src/import/engine.rs

```diff
@@ -214,6 +221,8 @@ impl SSTWriter {
     }
 }
 
+/// Get set of ranges for `props`, the maximum number of ranges can not exceed `max_ranges`,
```

Comment:
```suggestion
/// Gets a set of approximately equal size ranges from `props`. The maximum number of ranges cannot exceed `max_ranges`,
```

Reply:
Note: I add "and" because we cannot combine two simple sentences using a comma (,). We can use a semicolon (;) or a conjunction ("and", "or", "but", "because", "so", and so on) to combine two sentences.

Reply:
It's a very helpful tips and thank you very much. 😂